assets/theme-css/panel.css

```diff
@@ -30,8 +30,9 @@
     height: 140px;
     min-width: 275px;
     width: 335px;
-    margin: 25px auto;
+    margin: 25px 15px 25px 15px;
     border-radius: 3px;
+    box-shadow: 0px 8px 15px var(--colorShadow);
```

Comment:
I added it when I removed the panel-underline thing.  Now that it is back, I don't think this is needed.

Reply:
Done